xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -26,6 +26,17 @@ class CAddonInfo;
 typedef std::shared_ptr<CAddonInfo> AddonInfoPtr;
 typedef std::vector<AddonInfoPtr> AddonInfos;
 
+enum AddonDisabledReason
```

Comment:
Maybe this should be `enum class`, because those types in difference to C enums can be forward declared.

Reply:
Interesting. I just learned something new 😉